.github/workflows/test-and-publish-nightly.yml

```diff
@@ -121,7 +131,13 @@ jobs:
               name: ubuntu-24.04-arm
             java-version: 11
             java-distribution: corretto
-            image: "public.ecr.aws/async-profiler/asprof-builder-arm:latest"
+            image: public.ecr.aws/async-profiler/asprof-builder-arm:latest
```

Comment:
On Alpine, we need to test binaries that were built on Debian.

Reply:
91e17e201827dd96750e1a78fa8cf61fe1f8a0df